sync/upstream.py

```diff
@@ -443,7 +448,9 @@ def wpt_commits(git_gecko, first_commit, head_commit):
                                       paths=env.config["gecko"]["path"]["wpt"],
                                       reverse=True,
                                       max_parents=1)]
-    return [item for item in commits if not item.metadata.get("wptsync-skip")]
+    return [commit for commit in commits
+            if not commit.metadata.get("wptsync-skip") or
```

Comment:
@jgraham Where does `wptsync-skip` get set? I can't find any other occurrences.

Reply:
You can set it in a commit message if the commit shouldn't be udpatreamed for whatever reason.